dist/setup-appliance.sh

```diff
@@ -427,6 +427,28 @@ function ask {
 
 }
 ###############################################################################
+function check_service {
+    srv=$1
+    STATE=$(chkconfig $srv 2>/dev/null|awk '{print $2}')
+    if [[ $STATE == "" ]]; then
+        ENABLED=`systemctl is-enabled $srv`
+        if [[ "$ENABLED" == "enabled" ]];then
+            STATE=on
+            ACTIVE=`systemctl is-active $srv`
+            [[ "$ACTIVE" == "active" ]] || systemctl start $srv
+        fi
+    fi
+    if [[ $STATE != on ]];then
+      ask "Service $srv is not enabled. Would you like to enable it? [Yn]" "y"
+      case $rv in
+        y|yes|Y|YES)
+          systemctl enable $srv
+          systemctl start $srv
```

Comment:
`systemctl enable --now $srv`

Reply:
I added this switch to all systemctl enable calls, see commit  445bdc0